pkg/controller/monitor/monitor_controller.go

```diff
@@ -166,8 +165,8 @@ func (r *ReconcileMonitor) Reconcile(ctx context.Context, request reconcile.Requ
 	}
 
 	if !r.prometheusReady.IsReady() {
-		err = fmt.Errorf("waiting for Prometheus resources")
-		r.setDegraded(reqLogger, err, "Waiting for Prometheus resources to be ready")
+		err = fmt.Errorf("waiting for Prometheus resources and add watches")
```

Comment:
What are you trying to convey with this addition? I do not understand and I doubt the user would either.
I think maybe this would be better as `waiting for Prometheus resources to be ready`.

Reply:
Reverted.